.editorconfig

```diff
@@ -16,9 +16,9 @@ indent_style = space
 indent_size = 4
 continuation_indent_size = 4
 
-[*.{js,css,html}]
+[*.{js,ts,scss,css,html}]
 indent_style = space
-indent_size = 4
```

Comment:
pls return it back to 4. We work with a common guideline, where it is 4

Reply:
done